### DISCUSSION TITLE
Unable to add click event on dropdown type element on breeze excluded pages

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#176] [1. Help]
Hi,
I am unable to add Click event on dropdown type element on breeze excluded pages. 
Ex: I have My account element in header in which I have added a click event. It's working fine on breeze pages but not working breeze excluded pages.
Any solutions?

**Comment by vovayatsyuk** [comment]
When you exclude the page, breeze-js is not added. You need to implement the same js for this page (Just like you would do for any other Luma-based theme.)
